### ISSUE TITLE
[FEATURE REQUEST] Allow for `API#getAllContainingTemplates` to check non-viewed scenes

### ISSUE THREAD

**Issue by tirzah2** [krbz999/babonus#320]
Hi! I'm trying to check if a token is inside a template on a scene that i am not currently viewing. Basically to have babonus that works only if a party token (on the overland map) is inside a template, while the players are fighting in a separate scene. 

**Setup**
- Build-a-Bonus version: Latest
- Foundry core version and build: 2.3.x
- Tagger module

```js
const ilMondoScene = game.scenes.find(scene => scene.name === "Il Mondo");
if (!ilMondoScene) return false;

const partyToken = ilMondoScene.tokens.find(t => t.name === "Party");
if (!partyToken) return false;

const templateIds = babonus.getAllContainingTemplates(partyToken);

function checkIfHasFreddoTag(templateId) {
    const template = ilMondoScene.templates.get(templateId);
    return Tagger.hasTags(template, "freddo", { caseInsensitive: true });
}

return templateIds.some(checkIfHasFreddoTag);
```

**Comment by krbz999** [comment]
With impending changes coming in v12, best to mark this as blocked by core until then.

**Comment by krbz999** [comment]
Closing this in favor of simply using scene regions. This requires no modules to do:

```js
TokenDocument#regions
```
